The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/8852971898/job/24312780466) [(plain)](https://github.com/rust-lang/rust/commit/4df1303cff1eeb3b0c7f00d70fecf160279e7263/checks/24312780466/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
..............................................................i....

failures:

---- library/core/src/slice/raw.rs - slice::raw::from_raw_parts (line 92) stdout ----
error[E0599]: no method named `sum` found for reference `&[f32]` in the current scope
   |
20 |     data.sum()
20 |     data.sum()
   |          ^^^ `&[f32]` is not an iterator
   |
help: call `.into_iter()` first
   |
20 |     data.into_iter().sum()

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0599`.
For more information about this error, try `rustc --explain E0599`.
Couldn't compile the test.

failures:
    library/core/src/slice/raw.rs - slice::raw::from_raw_parts (line 92)

test result: FAILED. 4599 passed; 1 failed; 43 ignored; 0 measured; 0 filtered out; finished in 153.93s

error: doctest failed, to rerun pass `-p core --doc`
  local time: Fri Apr 26 19:14:19 UTC 2024
  network time: Fri, 26 Apr 2024 19:14:19 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/core/src/slice/raw.rs
+++ b/library/core/src/slice/raw.rs
@@ -106,7 +106,7 @@ use crate::ub_checks;
 ///         // SAFETY: see function docstring.
 ///         unsafe { slice::from_raw_parts(ptr, len) }
 ///     };
-///     data.sum()
+///     data.into_iter().sum()
 /// }
 ///
 /// // This could be the result of C++'s std::vector::data():

